components/mpas-ocean/src/tracer_groups/Registry_freshwaterTracers.xml

```diff
@@ -0,0 +1,215 @@
+	<nml_record name="tracer_forcing_freshwaterTracers">
+		<nml_option name="config_use_freshwaterTracers" type="logical" default_value=".false." units="unitless"
+			description="if true, the 'freshwaterTracers' category is enabled for the run"
+			possible_values=".true. or .false."
+		/>
+		<nml_option name="config_use_freshwaterTracers_surface_bulk_forcing" type="logical" default_value=".true." units="unitless"
+			description="if true, surface bulk forcing from coupler is added to surfaceTracerFlux in 'freshwaterTracers' category. True by default because the main function of this tracer group is tracking freshwater fluxes"
+			possible_values=".true. or .false."
+		/>
+		<nml_option name="config_use_freshwaterTracers_surface_restoring" type="logical" default_value=".false." units="unitless"
+			description="if true, surface restoring source is applied to tracers in 'freshwaterTracers' category"
+			possible_values=".true. or .false."
+		/>
+		<nml_option name="config_use_freshwaterTracers_interior_restoring" type="logical" default_value=".false." units="unitless"
+			description="if true, interior restoring source is applied to tracers in 'freshwaterTracers' category"
+			possible_values=".true. or .false."
+		/>
+		<nml_option name="config_use_freshwaterTracers_exponential_decay" type="logical" default_value=".false." units="unitless"
+			description="if true, exponential decay source is applied to tracers in 'freshwaterTracers' category"
+			possible_values=".true. or .false."
+		/>
+		<nml_option name="config_use_freshwaterTracers_idealAge_forcing" type="logical" default_value=".false." units="unitless"
+			description="if true, idealAge forcing source is applied to tracers in 'freshwaterTracers' category"
+			possible_values=".true. or .false."
+		/>
+		<nml_option name="config_use_freshwaterTracers_ttd_forcing" type="logical" default_value=".false." units="unitless"
+			description="if true, transit time distribution forcing source is applied to tracers in 'freshwaterTracers' category"
+			possible_values=".true. or .false."
+		/>
+		<nml_option name="config_use_freshwaterTracers_surface_value" type="logical" default_value=".false." units="unitless"
+			description="if true, surface value is computed for 'freshwaterTracers' category"
+			possible_values=".true. or .false."
+		/>
+	</nml_record>
+
+	<packages>
+		<package name="freshwaterTracersPKG" description="This package includes variables required to include freshwaterTracers."/>
+		<package name="freshwaterTracersBulkRestoringPKG" description="This package includes variables required to compute bulk restoring on the freshwaterTracers group."/>
+		<package name="freshwaterTracersSurfaceRestoringsPKG" description="This package includes variables required to compute surface restoring on the freshwaterTracers group."/>
```

Comment:
```suggestion
		<package name="freshwaterTracersSurfaceRestoringPKG" description="This package includes variables required to compute surface restoring on the freshwaterTracers group."/>
```
Added `s` typo causes an error when this tracer group is activated and tested with compass `global_ocean/Icos240/WOA23/performance_test/forward`, using gnu debug on perlmutter.

Reply:
With that change, and using `config_use_freshwaterTracers = .true.`, the simulation runs to completion.

Reply:
Thanks for catching that typo, @mark-petersen!